packages/cli/src/index.ts

```diff
@@ -347,9 +347,18 @@ program
       }\nTo alter these settings use the parameters '--max-fee-per-gas', '--max-priority-fee-per-gas', '--gas-limit'.`
     );
 
+    const signer = p.signers[0];
+    let contract;
+    let recipient;
```

Comment:
@noahlitvin  I didnt know how should I fill `contract`, `recipient` and is `contract.estimateGas.publish` right or not?